internal/queue/actioner/fake/fake.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright (c) 2025 Diagrid Inc.
+Licensed under the MIT License.
+*/
+
+package fake
+
+import (
+	"context"
+
+	"github.com/diagridio/go-etcd-cron/api"
+	"github.com/diagridio/go-etcd-cron/internal/api/queue"
+	"github.com/diagridio/go-etcd-cron/internal/api/stored"
+	"github.com/diagridio/go-etcd-cron/internal/counter"
+)
+
+type Interface interface {
```

Comment:
do we need this? shouldn't this implement actioner.Interface?

Reply:
Thanks, copy paste error